core/templates/pages/topics-and-skills-dashboard-page/skills-list/skills-list.directive.ts

```diff
@@ -18,7 +18,7 @@
 
 require('components/skill-selector/merge-skill-modal.controller.ts');
 require(
-  'components/skill-selector/skill-selector.directive.ts');
+  'components/skill-selector/skill-selector.component.ts');
```

Comment:
import not required.

Reply:
Going to wait as I'm curious about [this question](https://github.com/oppia/oppia/pull/11791#discussion_r587011631) before making all the changes.

Reply:
Ditto, leaving this here because of [this comment](https://github.com/oppia/oppia/pull/11791#discussion_r587057593)